src/parser/bindingDOM.ts

```diff
@@ -67,7 +67,7 @@ export class BindingExpression {
 }
 
 export class Document {
-	public constructor(public bindings: Binding[], public viewmodelReferences: ViewModelNode[], public bindingHandlerReferences: BindingHandlerImportNode[]) { }
+	public constructor(public rootBinding: Binding, public viewmodelReferences: ViewModelNode[], public bindingHandlerReferences: BindingHandlerImportNode[]) { }
```

Comment:
Is there a root binding? If not, could this be improved?

Reply:
Yes. All documents have a root
Excerpt from the `createDocument` function.
```typescript
const root = new Binding(new BindingName('root', undefined as unknown as Location), new BindingExpression('', undefined as unknown as Location))
```